bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/image/ImageCatalog.scala

```diff
@@ -2,85 +2,121 @@ package edu.gemini.catalog.image
 
 import java.net.URL
 
-import edu.gemini.spModel.core.{MagnitudeBand, Angle, Coordinates}
-import jsky.util.Preferences
+import edu.gemini.spModel.core.{Angle, Coordinates, MagnitudeBand, Wavelength}
+import edu.gemini.spModel.core.WavelengthConversions._
+
+import scalaz._
+import scalaz.NonEmptyList._
+import Scalaz._
 
 /** Represents an end point that can load an image for a given set of coordinates */
-sealed abstract class ImageCatalog(val id: String, val displayName: String) {
-  /** Returns the url that can load the passed coordinates */
-  def queryUrl(c: Coordinates): URL
+sealed abstract class ImageCatalog(val id: String, val displayName: String, val shortName: String) {
```

Comment:
It is a unique ID used in the file name to reparse them into files belonging to a given catalog. I guess it could be a `sealed trait` to limit what you put there